winsup/cygwin/uinfo.cc

```diff
@@ -543,6 +543,36 @@ cygheap_user::env_systemroot (const char *name, size_t namelen)
   return psystemroot;
 }
 
+const char *
+cygheap_user::env_tty_handles(const char *name, size_t namelen)
+{
+  static unsigned initialized;
+  static char buffer[128];
+
+  if (!initialized)
+    {
+      int fd;
+      char *p = buffer;
+      strcpy(p, " ");
+      p += strlen(p);
+      for (fd = 0; fd < 3; fd++)
+        if (isatty(fd))
+          {
+            cygheap_fdget cfd (fd);
```

Comment:
Ok. I was only looking at your diff, and as I was assuming that you've adapted to the existing coding style, and the dominant style in your diff is to not have an extra space before the parenthesis (a style that I personally also prefer), I was further assuming that the existing style also does not use that extra space. But I was wrong. Not only `spawn.cc` but also `uinfo.cc` uses that extra space. I think it's ugly, but strictly speaking I think we should  adapt to that style everywhere in your diff then.